NOTICE.txt

```diff
@@ -1157,7 +1156,7 @@ Copyright (c) 2006-2013 Alexander Chemeris
 Copyright (c) The Internet Society (2005).
 Copyright (c) The Internet Society (2006).
 copyright 2009-2013 Christopher M. Kohlhoff
-Copyright (c) 2016, Akamai Technolgies, Inc.

```

Comment:
This appears to be a typo (Google finds fewer than 500 hits for this spelling, although, admittedly at least one is in a proxy statement to the SEC, but even there the correct spelling is also present...).